js/src/logger.ts

```diff
@@ -2792,7 +2800,6 @@ export function newId() {
 export class SpanImpl implements Span {
   // `internalData` contains fields that are not part of the "user-sanitized"
   // set of fields which we want to log in just one of the span rows.
-  private internalData: Partial<ExperimentEvent>;
```

Comment:
Maybe move this comment to `logInternal`?

Reply:
can do